docs/CONTRIBUTING.md

```diff
@@ -88,8 +88,8 @@ See [implement-an-exercise-from-specification](https://github.com/exercism/docs/
 
 The structure of en exercise's directory is as follows (note the differing hyphen and underscore usage for directory names and file names respectively):
 
-```
-+-- <my-exercise>
+<pre>
++-- &lt;my-exercise&gt;
```

Comment:
Do you need to use `&lt;` below as well? The rich diff shows that it's messed up as-is.